_minutes/2018-10-25.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2018-10-25 6pm
+---
+
+**Present:**
+Findlay Smith, Rokas Gudaviskus, Mo Javad, Joao Maio, Patrick Kage, Tony Duggan, Likhitha Sai, Kyle Cotton, Elena Lape
+
+**Ceilidh and InfBall**
+- Looking at Venues for the InfBall
+    - Neil Heatley has a list of approved discount venues that we should look into.
+- Ceilidh should have a burns supper added on to it.
+- Findlay will be talking to the Haggis.
+- Ceilidh would be Late-Jan/Early-Feb.
+
+**Argos Pub Quiz**
+- Event date either 9th or 6th of November.
+- Venue is Bar50.
+- Send out the event in a newsletter  to advertise.
+- Advertise on the Facebook group and page.
+- Marketing aimed at students in penultimate year.
+
+**Local Hack Day**
+- Taking place on Saturday from 10:30AM - 11PM.
+- Start resources page is up.
+    - Page includes back-end, front-end, ML, blockchain, etc.
+    - Will include lists of Open Source projects that students can work on.
+    - Patrick needs to add extra resources.
+    - Possibly put some things for inspirations of students.
+
+**Hack The Burgh**
+- Committee will be Athiya, Alan, Viktor, Findlay, Elena, Bianca(possible), Vivek, Sanvith(possible).
+- Union Of Genius has blacklisted us because of miscommunication on our part last year.
+- We are also not going with University Caterers.
+- JPMorgan and SquarePoint Capital have become our gold sponsors.
+
+**Business in Tech**
+- Speeches from Microsoft and Accenture.
+- A panel being moderated by Don Sannella.
+- We have decided to give glasses of wine to attendees.
+
+**NewsBytes and CompSoc Website**
+- Suggested by Caterina in last year's election.
+- Try and get it up and running at the beginning of the semester.
+- Thanks Joao.
+- The website will need to be redeveloped.
+    - Visual Overhaul
+    - Content and layout changes.
```

Comment:
It's easy to say "needs content changes" without identifying such changes! Pull requests are welcomed.

I agree it needs a visual overhaul.